src/components/utah-design-system/Accordion.jsx

```diff
@@ -0,0 +1,37 @@
+import * as Accordion from '@radix-ui/react-accordion';
+import PropTypes from 'prop-types';
+
+export function AccordionRoot({ type, children }) {
+  return <Accordion.Root type={type}>{children}</Accordion.Root>;
+}
+
+AccordionRoot.propTypes = {
+  type: PropTypes.oneOf(['single', 'multiple']),
+  children: PropTypes.node.isRequired,
+};
+
+export function AccordionPanel({ title, children }) {
+  return (
+    <Accordion.Item value={title} className="mb-1 overflow-hidden rounded-md">
+      <Accordion.Header className="rounded-sm">
+        <Accordion.Trigger className="group w-full justify-between rounded-b-none rounded-t-md border-[--primary-color] bg-[--primary-color] px-2 py-1 text-left text-lg leading-5 text-white active:transform-none active:shadow-none">
```

Comment:
**suggestion**: add these css variable values to tailwind so they will work if we decide to remove the design system css.